### PR TITLE
Shift to null safety

### PR DESCRIPTION
**Pull request by BrianMwas**
Changed the forked library to use latest null safety syntax

### CODE REVIEW

File: pubspec.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ version: 0.3.2
 homepage: https://github.com/flutter-webrtc/callkeep
 
 environment:
-  sdk: '>=2.7.0 <3.0.0'
+  sdk: '>=2.16.0 <3.0.0'
```
Comment (kmurtaja) [inline pubspec.yaml]: @BrianMwas I believe it should be `2.16.2` NOT `2.16.0`
What do you think?

**Review by kmurtaja — APPROVED** [review]

File: pubspec.yaml (hunk 1 above)
Comment (BrianMwas) [reply pubspec.yaml]: Sure I'll change it

**Review by kmurtaja — APPROVED** [review]
@BrianMwas 🔥 🚀

**Review by AshirShaikh99 — APPROVED** [review]
Champion of the programming

**Pull request merged**